### DISCUSSION TITLE
Changing weights file

### DISCUSSION THREAD

**Discussion by Allesweter** [LeelaChessZero/lc0#1847] [Q&A] [answered]
I just can't figure out how to change weights file of the engine on Android. How do i do that

**Comment by mooskagh** [comment, marked as answer]
This worked for some people:

* Download a network using into the standard Android Downloads directory, remember the filename.
* Set `WeightsFile` UCI parameter (how to configure it, depends on the Chess GUI) to `/storage/emulated/0/Download/(filename)`.

Hope this helps. If not, I'd recommend to go to the #help channel of our Discord chat at http://lc0.org/chat, it's usually much faster to troubleshoot there.

**Reply by Allesweter** [reply]
Tysm! I was just pasting the name of file lol

**Comment by DryStrike** [comment]
The latest Cuda improvement released 13 hours ago or so is causing the engine to crash in Fritz GUI
